elastic/kibana
# [Uptime] Prevent UI Monitor Management bottom bar from overlapping content

## Summary

Resolves https://github.com/elastic/kibana/issues/120808

Prevents UI monitor management bottom bar from overlapping content by moving the `ActionBar` component to the `bottomBar` prop on the page template. This ensures that the bottom bar has context surrounding the page layout, preventing it from overlapping any content. This is achieved by using portals.

Before
<img width="1755" alt="Screen Shot 2021-11-30 at 9 02 06 AM" src="https://user-images.githubusercontent.com/11356435/145859810-c5e8e710-cd56-4e70-9a8b-205cbf764e01.png">

After
<img width="1868" alt="Screen Shot 2021-12-13 at 12 23 19 PM" src="https://user-images.githubusercontent.com/11356435/145859835-7f89dab2-a52c-4a5d-8bf9-601b68f10071.png">
<img width="1872" alt="Screen Shot 2021-12-13 at 12 23 09 PM" src="https://user-images.githubusercontent.com/11356435/145859842-93df17e6-cba5-4e3a-93eb-e6616ac88939.png">

## How to test this PR locally

- Go through [this](https://gist.github.com/awahab07/8456a2b1854ed9c5dcd30fb4e733ae1c) and enable monitor management.
- Add or Edit a monitor and confirm the PR implements the said behavior.
